src/js/AuthUI.js

```diff
@@ -0,0 +1,17 @@
+export default class Auth {
+  static user;
+
+  static pass;
+
+  static error = document.querySelector('#auth-error');
+
+  static main = document.querySelector('#main');
+
+  static regExpPass = new RegExp(
+    /^(?=.*[a-zA-Z])(?=.*[0-9!@#$%^&*?+])(?!.*[()_-`\\/"'|[]}{:;>.<,])(?!.*\s)(?!.*\s).{8,20}$/
```

Comment:
I think it's better just to check is string empty

Reply:
as you wish